portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+const pictures = ["images/proj1.jpg","images/proj2.jpg","images/proj3.jpg"]
+var currentIndex = 0;
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function nextImage(){
+    if(currentIndex < pictures.length - 1){
+        currentIndex ++;
+    }
+    else{
+        currentIndex = 0;
+    }
```

Comment:
While this works, a (potentially cleaner) alternative is to increment and then use mod division to get the new index. Up to you which way you like better.

Reply:
I realized that this code could actually be done better. I fixed this in my update.